description: >
  Testing exceptions
runExportedFunction: 0
expectException: "My uncaught exception"
testOnly: true
expectedPrintout: foo
assertionCount: 7